autometa/common/external/diamond.py

```diff
@@ -297,8 +301,7 @@ def blast(
     cpus : int, optional
         Number of processors to be used, by default uses all the processors of the system
     tmpdir : str, optional
-        Path to temporary directory
-        '</path/to/temporary/directory>' by default output directory
+        Path to temporary directory by default, output directory
```

Comment:
Should be two sentences.

```suggestion
        Path to temporary directory. By default, same as the output directory
```

Reply:
I was using VS code extension for docstring with the type set to `numpy`. Going through `numpy` docstring documentation I was not able to find a standard way to mention the default value, but since we all are now using VS code with the same extension, I think having "by default" in the same sentence would help maintain consistency. What do you think about this?

Reply:
No. I put this suggestion because the sentence structure you put is wrong. "Path to temporary directory by default" means that by default it is a temporary directory, and I suppose if the user chooses it will be some other type of directory? It doesn't make much sense. Then there is the other part of the sentence. "Path to temporary directory by default, output directory". The "output directory" part is out there on its own as if it was the start of the next clause, but then there is no verb or anything else. The punctuation I suggested makes it unambiguous that "By default" belongs with "output directory". I think your priority should be clarity before consistency. (But also I highly doubt that the numpy docstrings are grammatically wrong).

Reply:
Please address this before we merge the PR.